### DISCUSSION TITLE
[CUSA16164] Sniper Elite 5 1.06

### DISCUSSION THREAD

**Discussion by pcfreaxx** [GoldHEN/GoldHEN_Cheat_Repository#40] [Cheat requests]
Poorly i am not able to create Cheats. I would be happy when some nice Guy create some. Maybe invisibility ;-)
If i know how to do it, i would help.

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
